var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -255,6 +255,14 @@ def configure_args(self):
             else:
                 config_args.append('--disable-mpi-thread-multiple')
 
+        # CUDA support
+        if spec.satisfies('@1.6:'):
+	    if '+cuda' in spec:
+                config_args.append('--with-cuda={0}'.format(spec['cuda'].prefix))
+                config_args.append('--with-cuda-libdir={0}/lib64'.format(spec['cuda'].prefix))
```

Comment:
is it always `lib64`? in the past we had various issues where hardcoding `lib` or `lib64` lead to issues.
If `cuda` contains a single library named `libcuda.so` or alike, then the default interface to `libs` by @alalazo should work and you would be able to do `spec['cuda'].libs.directories` to get what you need.